CHANGELOG.rst

```diff
@@ -5,6 +5,30 @@ Dell OpenManage Ansible Modules Release Notes
 .. contents:: Topics
 
 
+v8.5.0
+======
+
+Release Summary
+---------------
+
+- Ansible lint issues are fixed for Redfish roles.
```

Comment:
Ansible lint issues are fixed for the collections

Reply:
done